devel/Dockerfile

```diff
@@ -57,6 +57,17 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
+# apt doesn't provide qemu-system-arm that support running armv8r52-mps3an536 target
+# install qemu-system-arm v9.0.2 to run armv8r52-mps3an536 target
+RUN git clone https://github.com/qemu/qemu.git --branch v9.0.2 && \
```

Comment:
use ARG for specifying the target version (easier to update the docker image in the future)